RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3617,14 +3658,14 @@ boolean auto_change_mcd(int mcd, boolean immediately)
 			return false;
 		}
 	}
-	if($strings[Blender, Packrat, Wombat] contains my_sign())
+	if(inGnomeSign())
 	{
-		if(get_property("lastDesertUnlock").to_int() < my_ascensions())
+		if(gnomads_available())
```

Comment:
This check should be inverted and the outer if removed.

Reply:
You are right, inverted it.

But left the outer if. if that one was removed it will return false on knoll sign and canadia sign. We only want it to return false if we are in a gnome sign and gnomads are not available